java-checks/src/test/files/checks/UselessImportCheck/WithPackage.java

```diff
@@ -54,3 +58,14 @@ void test() throws MyException, MyException2 {
   }
   void foo(@Nullable int x){}
 }
+
+class A {
+  byte @MyAnnotation2 [] table = null;
```

Comment:
`byte @MyAnnotation2 [] @MyAnnotation4 [] table = null `
Are we sure that annotations on all dimensions are correctly detected ?